engine/table/src/main/java/io/deephaven/engine/table/impl/UpdateByControl.java

```diff
@@ -0,0 +1,112 @@
+package io.deephaven.engine.table.impl;
```

Comment:
Is UpdateByControl something we want to be able to have clients be able to set via gRPC eventually? If so, this should move out of the engine impl and into the table-api (like other arguments to updateBy).